src/tracking/config.py

```diff
@@ -0,0 +1,12 @@
+upstreams = {
```

Comment:
That is the plan, but it will need some work and careful thought. So, we'll get this in, in a different PR. I plan to scan the containers and save their enabled repos to db. Then, a modified version of this script goes and fetches the update/change data.